modes/fr/scan.md

```
@@ -0,0 +1,242 @@
+   g. Accumuler dans la liste de candidats (dédup avec Niveau 1)
+
+6. **Niveau 3 — Requêtes WebSearch** (parallèle si possible) :
+   Pour chaque query dans `search_queries` avec `enabled: true` :
+   a. Exécuter WebSearch avec le `query` défini
+   b. De chaque résultat extraire : `{title, url, company}`
+      - **title** : du titre du résultat (avant le « @ » ou « | »)
+      - **url** : URL du résultat
+      - **company** : après le « @ » dans le titre, ou extraire du domaine/path
+   c. Accumuler dans la liste de candidats (dédup avec Niveau 1+2)
+
+6. **Filtrer par titre** en utilisant `title_filter` de `portals.yml` :
```

markdownlint-cli2 (0.22.0)

[warning] Ordered list item prefix
Expected: 7; Actual: 6; Style: 1/2/3

(MD029, ol-prefix)